charts/dremio_v2/config/logback.xml

```diff
@@ -16,7 +16,7 @@
     limitations under the License.
 
 -->
-<configuration>
+<configuration scan="true" scanPeriod="30 seconds">
```

Comment:
This will cause the Dremio pods to restart when this is deployed - so let's wait on this.